### DISCUSSION TITLE
ghippo/best-practice/navigator/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3636] [Announcements]
# ghippo/best-practice/navigator/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/best-practice/navigator/

**Comment by samzong** [comment]
请问在什么版本之后开始支持 `GProductNavigator`  这个 CRD ？

**Reply by yitingdc** [reply]
这个CRD从很早的版本就开始支持了

**Reply by windsonsea** [reply]
看 [全局管理 Release Notes](https://docs.daocloud.io/ghippo/intro/release-notes/#2022-09-28) 最早 v0.10.0 有提及 GProduct
